app/javascript/custom-tagged-plate/components/CustomTaggedPlate.spec.js

```diff
@@ -43,106 +43,106 @@ describe('CustomTaggedPlate', () => {
         expect(wrapper.vm.isChildWellsValid).toEqual(false)
       })
 
-      it('returns false if any wells with aliquots are invalid', () => {
+      it('returns false if any wells with aliquots are invalid', async () => {
```

Comment:
So when is async required (with await)? Is it best practice anytime you set data? Does that give it chance to run the computed functions etc. before your expect?

Reply:
This was one of the changes that came out of the vue-test-utils update (sorry to base off that, but I didn't want to migrate the test suite with an outdated library that I knew was causing failures).

But yeah, all the vue documentation examples use await on both setting data and props. Most of these changes were practical (ie. they actually fixed a new failure) rather than theoretical, but I don't really see reason to skip it.